src/without-bang-important.js

```diff
@@ -0,0 +1,23 @@
+// Module with the same interface as the core aphrodite module,
+// except that styles injected do not automatically have !important
+// appended to them.
+//
+import {injectAndGetClassName} from './inject';
+
+import {
+    StyleSheet,
+    StyleSheetServer,
+    StyleSheetTestUtils,
+} from './index.js'
+
+const css = (...styleDefinitions) => {
+    // Do not append !important to style definitions
+    return injectAndGetClassName(false, styleDefinitions);
```

Comment:
I see your intent, but I think the comment is more descriptive than that variable name, and it ends up being the same # of lines. If this thing had many arguments of ambiguous meaning, I'd be more inclined to the go the variable naming route (possibly still keeping the comments above the variable declarations.